inductiva/commands/commands.py

```diff
@@ -0,0 +1,8 @@
+"""Wrapper class for simulator commands."""
+
+
+class Command(dict):
+    """Abstraction class for commands."""
+
+    def __init__(self, cmd, *prompts):
+        super().__init__(cmd=cmd, prompts=list(prompts))
```

Comment:
no need to inherit from dict because the class then inherits a lot of methods that do not make sense to exist in this class.

Make it a simple class:


```suggestion
class Command(dict):
    """Abstraction class for commands."""

    def __init__(self, cmd, *prompts):
        self.cmd=cmd
        self.prompts=prompts # <-- keep it a tuple
    
    def to_json(self):
        return self.__dict__ # or just return a dic with the required keys
```


and to make it json serializable, use something like:

```python
>>> c = Command('command', 'p1', 'p2')
>>> json.dumps(c, default=lambda x: x.to_json())
{'cmd': 'command', 'prompts': ('p1', 'p2')}
```